frontend/src/utilities/apiCalls.ts

```diff
@@ -0,0 +1,2 @@
+import React from "react"
+
```

Comment:
Whats this empty file for?

Reply:
I was gonna move stuff into it but you can remove it